app/views/Project/Tagging/Assessments/ActionCell/index.tsx

```diff
@@ -84,12 +90,22 @@ function ActionCell(props: Props) {
         }
     }, [deleteAssessment, projectId, assessmentId]);
 
+    const assessmentEditLink = useMemo(() => ({
+        pathname: generatePath(
+            routes.assessmentEdit.path,
+            {
+                projectId,
```

Comment:
We do not need to pass projectId in most of the cases. It is automatically injected